src/mapTools.ts

```diff
@@ -106,7 +106,7 @@ function getInitialBounds(
     locations.push(cornerEastSouth)
   }
 
-  const bounds = new window.L.latLngBounds()
+  const bounds = window.L.latLngBounds()
```

Comment:
@keyy123 @ojafero do you maybe know what parameters this function needs? I get a `Expected 1-2 argument, but got 0`

Reply:
I looked through the source code. The error I get keeps saying window && globaThis does not have prop 'L' so I declare a global with windows as L. As for latLngBounds, I searched through the code base in visual code and the latLngBounds method is only declared within map tools. Maybe it's legacy code or it's unfinished? I read the map.ts file and it looks like maybe the getBounds method is its successor. 🦝

Reply:
@RubenSmn  In short, I don't know the parameters because the function does not exist?

Reply:
@keyy123 the `window.L` should be fixed, maybe run `npm install` to add the missing types for `leaflet`? Hmm I guess I'll undo my changes on this line and leave it for now.

Reply:
https://www.loom.com/share/d1ed4344767e421cb64e5f9f672cfbea

[I found out how to solve the argument issue]

Reply:
Haha thanks @keyy123 I filled them with "empty" coordinates.